doc/rst/technotes/local.rst

```diff
@@ -39,7 +39,7 @@ The ``local`` construct does not necessarily indicate the cause of
 communication when present. See the `Module\: CommDiagnostics`_ for ways to
 diagnose communication.
 
-.. _Module\: CommDiagnostics:    https://chapel-lang.org/docs/latest/modules/standard/CommDiagnostics.html
+.. _Module\: CommDiagnostics:    https://chapel-lang.org/docs/modules/standard/CommDiagnostics.html
```

Comment:
@ben-albrecht : For each release, should this be pointing to that releases current version of the CommDiagnostics documentation rather than the "latest" version of that documentation?

Reply:
Yes, but we shouldn't be cross-referencing documentation with full URLs like this in the first place. This should be accomplished with a sphinx cross-reference, in this case a [Chapel domain module reference](https://sphinxcontrib-chapeldomain.readthedocs.io/en/latest/reference.html#role-chpl:mod):

```rst
See the :chpl:mod:`Module: CommDiagnostics <CommDiagnostics>` for ways to diagnose communication.
```

Reply:
If only there were some way to know who injected this in the first place...  :)